workflows/edit_eml/edit_semantic_annotation.Rmd

```diff
@@ -39,17 +55,68 @@ $valueURI$valueURI
 [1] "http://purl.dataone.org/odo/ECSO_00002617"
 ```
 
-### Ontologies used
+### How to add an annotation
+
+**1. Decide which variable to annotate**
+
+The goal for the datateam is to start annotating every dataset that comes in. Practically, we cannot annotate
+every single variable in every single dataset - this would take forever! Typically, there are a few key variables
+that are measured in any particular dataset. These key variables are the ones we want to prioritize. Although having annotations on 
+variables like date, longitude, or site name would be helpful, they shouldn't be our primary focus.
+
+
+**2. Find an appropriate `valueURI`**
+
+The next step is to find an appropriate value to fill in the blank of the sentence: "this attribute contains measurements of _____."
+
+There are several ontologies to search in. In order of most to least likely to be relevant to the Arctic Data Center they are:
+
+* [The Ecosystem Ontology (ECSO)](http://bioportal.bioontology.org/ontologies/ECSO/?p=classes&conceptid=http://purl.dataone.org/odo/ECSO_00002925)  
+    - this was developed at NCEAS, and has many terms that are relevant to ecosystem processes, especially those involving carbon and nutrient cycling
+* [The Environment Ontology (EnVO)](http://bioportal.bioontology.org/ontologies/ENVO/?p=classes&conceptid=root)
+    - this is an ontology for the concise, controlled description of environments
+* [National Center for Biotechnology Information (NCBI) Organismal Classification (NCBITAXON)](http://bioportal.bioontology.org/ontologies/NCBITAXON/?p=classes&conceptid=root)
+    - The NCBI Taxonomy Database is a curated classification and nomenclature for all of the organisms in the public sequence databases.
+* [Information Artifact Ontology (IAO)](http://bioportal.bioontology.org/ontologies/IAO/?p=summary)
+    - this ontology contains terms related to information entities (eg: journals, articles, datasets, identifiers)
+
+To search, navigate through the "classes" until you find an appropriate term.
```

Comment:
```suggestion
To search, navigate through the "classes" until you find an appropriate term. When we are picking terms, it is important that we not just pick a similar term or a term that seems close -  we want a term that is totally "right". For example, if you have an attribute for carbon tetroxide flux and an ontology with a class hierarchy like this:

-- carbon flux
  |---- carbon dioxide flux

Our exact attribute, carbon tetroxide flux is not listed. In this case, we should pick "carbon flux" as it's completely correct and not "carbon dioxide flux" because it's more specific but not quite right.
```